pilot/pkg/model/push_context.go

```diff
@@ -1760,8 +1760,8 @@ func (ps *PushContext) initWasmPlugins(env *Environment) error {
 
 	sortConfigByCreationTime(wasmplugins)
 	ps.wasmPluginsByNamespace = map[string][]*WasmPluginWrapper{}
-	for _, plugin := range wasmplugins {
-		if pluginWrapper := convertToWasmPluginWrapper(&plugin); pluginWrapper != nil {
```

Comment:
seems no problem here, WasmPluginWrapper does not keep the reference of plugin

Reply:
OK, I'll revert it.